scrapper.py

```diff
@@ -26,36 +31,66 @@ class Crawler:
     Crawler implementation
     """
     def __init__(self, seed_urls, max_articles: int):
-        pass
+        self.seed_urls = seed_urls
+        self.max_articles = max_articles
+        self.urls = []
 
     def _extract_url(self, article_bs):
-        pass
+        class_bs = article_bs.find('div', class_="b-materials-list b-list_infinity")
+        title_bs = class_bs.find_all('p', class_="b-materials-list__title b-materials-list__title_news")
+        for link in title_bs:
+            link = link.find('a')
+            self.urls.append('https://magadanpravda.ru/' + link['href'])
 
     def find_articles(self):
         """
         Finds articles
         """
-        pass
+        for urls in self.seed_urls:
+            response = requests.get(urls)
+            response.encoding = 'utf-16'
+            with open('index.html', 'w', encoding='utf-16') as file:
```

Comment:
This code is going to create a html file in the directory where scrapper.py is located.
We do not need to save all the visited pages as html.
Refer to `docs/datset.md` to see where and how data must be stored.